gym-service/src/main/java/org/tolking/controller/client/TrainingEventClient.java

```diff
@@ -0,0 +1,13 @@
+package org.tolking.controller.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.tolking.config.FeignClientConfig;
+import org.tolking.external_dto.TrainingEventDTO;
+
+@FeignClient(name = "training-tracker-service", configuration = FeignClientConfig.class)
+public interface TrainingEventClient {
+    @RequestMapping("/event")
```

Comment:
I don't have strong preferences, but code will look clearer if we split methods with empty line from each other, variables and class definitions